File: leetcode/easy/1672-richest-customer-wealth/solution.java
class Solution {
    public int maximumWealth(int[][] accounts) {
       int maxVal=Integer.MIN_VALUE;
       int rows=accounts.length;
       int cols=accounts[0].length;
       for(int i=0;i<rows;i++){
            int sum=0;
            for(int j=0;j<cols;j++){
                sum += accounts[i][j];
            }
            maxVal=Math.max(maxVal,sum);
       }
       return maxVal;
    }
}

File: leetcode/easy/1672-richest-customer-wealth/solution.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int rowSize = accounts.size();
        int colSize = accounts[0].size();

        int maxWealth = INT_MIN;
        for (int i = 0; i < rowSize; i++) {
            int sumn = 0;
            for (int j = 0; j < colSize; j++) {
                sumn += accounts[i][j];
            }
            maxWealth = max(maxWealth, sumn);
        }

        return maxWealth;
    }
};